packages/inventory/src/loaders/container.ts

```diff
@@ -0,0 +1,20 @@
+import { InternalModuleDeclaration, LoaderOptions } from "@medusajs/modules-sdk"
+
+import {
+  InventoryItemService,
+  InventoryLevelService,
+  ReservationItemService,
+} from "../services"
+
+import { asClass } from "awilix"
+
+export default async (
+  { container }: LoaderOptions,
+  moduleDeclaration?: InternalModuleDeclaration
+): Promise<void> => {
+  container.register({
+    inventoryItemService: asClass(InventoryItemService),
```

Comment:
todo: Specify a lifetime otherwise it will be transient by default and be instantiated every time they are resolved. unless it is wanted

Reply:
we are only setting `.singleton` for `asFunction` on the entire core. Should we add that to all `asClass` as well? in this case is ok, but not sure about the rest

Reply:
I have a ticket to allow to specify the lifetime on the services. but the core services are always registered as singleton. But the plugins services lifetime are not specified and will be part of the ticket I mentioned.

Otherwise, it will create plenty of instances each time and since the injected deps are transient it will re instanciate them and so on. Which create overhead on each request